enterprise/__init__.py

```diff
@@ -4,6 +4,6 @@
 
 from __future__ import absolute_import, unicode_literals
 
-__version__ = '0.1.2'
+__version__ = '0.1.3'
```

Comment:
@zubair-arbi @mattdrayer we haven't decided on the versioning yet, but so far we more or less adhered to "continuous integration" approach - releasing new version as soon as it is merged. This app is not used in production though, so we might change the approach later, but so far I would suggest keeping the "aggressive" releasing strategy as long as we can. In that case, every PR is a new minor version, and it is released to edx-platrofm right after it is tagged.